lib/tasks/migrate_data.rake

```diff
@@ -99,4 +99,36 @@ namespace :tww do
       end
     end
   end
+
+  desc "migrate some test data from quickbooks"
+  task migrate_organizations_to_qbo: :environment do
+    qb_connection = Division.root.qb_connection
+
+    customers = Quickbooks::Service::Customer.new(qb_connection.auth_details).all
+
+    customer_hash = {}
+    customers.each do |customer|
+      # Customer does not have a name property so we use display name since that is
+      # what is usually show in the QBO UI.
+      customer_hash[customer.display_name] = customer
+    end
+
+    Organization.all.find_each do |org|
+      customer = customer_hash[org.name]
+
+      if customer
+        if org.qb_id.blank?
+          puts "Mapping organization '#{org.name}' to #{customer.id}"
+          org.update!(qb_id: customer.id)
+        else
+          if org.qb_id != customer.id
+            puts "ERROR! 'Problem mapping customer #{customer.id} to Coop #{org.name}', it is already mapped to #{org.qb_id}, skipping"
+            next
+          end
+        end
+      end
```

Comment:
Should we have an `else` block in which we output a message that no matching Org was found for customer xxx?